rest-api-spec/src/main/resources/rest-api-spec/api/indices.delete.json

```diff
@@ -20,6 +20,11 @@
         "master_timeout": {
           "type" : "time",
           "description" : "Specify timeout for connection to master"
+        },
+        "include_hidden": {
```

Comment:
ok good to know, maybe those are considered default params by the clients, not sure.